tests/unit_tests/core/test_context.py

```diff
@@ -13,9 +14,17 @@
     Stoppable,
     SyncOrAsync,
 )
+from bluesky.run_engine import RunEngine
 from bluesky.utils import MsgGenerator
 from dodal.common import PlanGenerator, inject
-from ophyd.sim import SynAxis, SynGauss
+from ophyd_async.core import (
```

Comment:
we should probably have ophyd-async in our dev dependencies if we rely on it directly